public/js/action-list.js

```diff
@@ -28,194 +28,616 @@
 
             this.on('click', '.action-list [data-action-item]:not(.page-separator), .action-list [data-action-item] a[href]', this.onClick, this);
             this.on('close-column', this.onColumnClose, this);
+            this.on('column-moved', this.onColumnMoved, this);
 
             this.on('rendered', '.container', this.onRendered, this);
```

Comment:
Should use `#main > .container` as selector. Otherwise the nested container check in the listener leads to false positives.